README.md

```diff
@@ -48,36 +48,32 @@ Examples of Data broker platforms are ARC, Galaxy, ...
 
 ### MARS-CLI
 
-This command line tool (CLI) will be used by the Data broker platform and will perform the actual submission of the ISA-JSON to the repositories. The application will be build as a Python library which can be integrated in a web application, ARC, Galaxy and others. Source code and documentation can be found in the [mars_cli folder](/) in this repo.
+This command line tool (CLI) will be used by the Data broker platform and will perform the actual submission of the ISA-JSON to the repositories. Based on receipts repositories give back as response, the ISA-JSON will be updated with accession numbers. The application is build as a Python library which can be integrated in a web application, ARC, Galaxy and others. Source code and documentation can be found in the [mars-cli folder](/mars-cli/) in this repo.
 
 The main steps of MARS-CLI are:
 
-1. **Validating the ISA-JSON**: Syntax validation
+1. **Ingesting and validating the ISA-JSON**: Compared to the vanilla ISA specification, the MARS-CLI has certain fields required (for example `target repository`) in order to function properly. Upon ISA-JSON ingestion the information gets loaded in the
```

Comment:
Is there something missing at the end here?

Reply:
Woopsy you are right! Thanks for spotting